medusa/process_tv.py

```diff
@@ -453,26 +457,17 @@ def unrar(self, path, rar_files, force=False):
                             break
 
                     if not skip_extraction:
-                        rar_handle.extract(path=path, withSubpath=False, overwrite=False)
+                        rar_handle.extract(path)
```

Comment:
how not ? 
 `While with unrar2 subfolders were discarded (withSubpath=False) this is not the case anymore` : there is no subfolder in a rared torrent

about extract, its inside a a for-loo: `for archive in rar_files:` so we only have one file